datadog_lambda/trigger.py

```diff
@@ -288,7 +288,7 @@ def extract_http_tags(event):
     """
     Extracts HTTP facet tags from the triggering event
     """
-    http_tags = {}
+    http_tags = {"span.kind": "server"}
```

Comment:
Good catch, I originally made [changes](https://github.com/DataDog/datadog-lambda-python/pull/635) to add this tag to all the related synthetic spans but missed it on the actual `aws.lambda` span. Is there a plan on the roadmap for getting this done in the Node Layer and the Rust extension as well?